app/strings.php

```diff
@@ -0,0 +1,1365 @@
+            __('No, I’m interested in a cross-disability group (includes disability, Deaf, and supporters)'),
+            __('Yes, I’m interested in a specific disability or Deaf group'),
+            __('Other experiences or identities'),
+            __('No, give me a group with intersectional experiences and/or identities'),
+            __('Yes, I’m looking for a group with a specific experience and/or identity (ex. Indigenous, immigrant, 2SLGBTQIA+)'),
+            __('Summary'),
+            __('Disability or Deaf groups'),
+            __('Other identities'),
+            __('Intersectional'),
+            __('Save'),
+        ],
+        // https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=2602%3A65596
+        'Manage project/Participant selection criteria/specific disability' => [
+            __('Select specific provinces or territories'),
+            __('Select all'),
+            __('Select none'),
+            __('Select specific cities or towns'),
+            __('Yes, I’m interested in a specific disability or Deaf group or groups'),
+            __('What specific disability and Deaf group or groups are you interested in engaging?'),
+            __('Visual including sight loss, blind, and partially sighted'),
+            __('Deaf'),
+            __('Pain-related'),
+            __('(such as chronic fatigue syndrome)'),
+            __('(includes individuals with no spoken or signed language who communicate using gestures, pictures, letter boards, communication devices or assistance from a person who knows them well)'),
+            __('(such as Autism, ADHD)'),
+            __('(includes intellectual disability)'),
+            __('(such as dyslexia)'),
+            __('(includes traumatic brain injury, memory difficulties, dementia)'),
+            __('Mental health-related'),
+            __('(such as dual diagnosis of a mental health barrier, substance dependence)'),
+            __('(such as environmental, HIV, migraine)'),
+            __('Is there a group with a specific experience of identity you are interested in engaging?'),
+            __('Types of experiences or identities'),
+            __('Age'),
+            __('Gender and sexual identity'),
+            __('Indigenous'),
+            __('Race and ethnicity'),
+            __('Immigrants and/or refugees'),
+            __('First language'),
+            __('Living in urban, rural, or remote areas'),
+            __('Notice'),
+            __('Please note that if you want to reach a specific identity group, you will not be able to use our matching service. You can reach them through an open call project, or go through a Community Connector or community organization.'),
+            __('Visual (including sight loss, blind, and partially sighted)'),
+            __('Women'),
+            __('Non-binary, gender non-conforming, gender fluid'),
```

Comment:
These are already in `DisabilityTypeSeeder` and `GenderIdentitySeeder` (the last one is in `resources/views/organizations/edit/2.blade.php`)